features/step_definitions/back_office/cancel_steps.rb

```diff
@@ -0,0 +1,13 @@
+When("I cancel the registration") do
+  visit_registration_details_page(@reg_number)
```

Comment:
Should that be
```suggestion
  visit_registration_details_page(@pending_reg_number)
```
?

Reply:
No, the only step that sets pending_reg_number is the one created for the permissions checks. All others, still use the normal @reg_number